app/models/rescan_runner.rb

```diff
@@ -95,13 +95,21 @@ def process_file(location, codec, path)
       end
 
       albumartist = Artist.find_by(name: t_albumartist) || Artist.new(name: t_albumartist, review_comment: 'New artist')
+      albumartists = if t_albumartist.downcase != 'various artists'
+                       [AlbumArtist.new(artist: albumartist,
+                                        name: t_albumartist,
+                                        order: 1,
+                                        separator: nil)]
+                     else
+                       []
+                     end
+
+      album = Album.find_by(title: t_album, release: Date.ordinal(t_year)) || Album.new(title: t_album,
```

Comment:
Is there a reason you moved the `Album.new` to this line? Because it's a lot more readable on the next, IMO.

Reply:
Have corrected this. Think this was rubocop's doing